cleanlab/count.py

```diff
@@ -199,6 +202,44 @@ def calibrate_confident_joint(confident_joint, labels, *, multi_label=False) ->
     return round_preserving_row_totals(calibrated_cj)
 
 
+def _calibrate_confident_joint_multilabel(confident_joint: np.ndarray, labels: list) -> np.ndarray:
```

Comment:
I presume for all the args of this function, only their multilabel versions are supported?  That should be clarified for each arg below, so you should say: 

"See `calibrate_confident_joint` documentation for more information (only the multilabel format of this argument is supported here)"

Note above I am also using more formal language (not statements like "docstring")

Reply:
I've added these statements in the required places.